h/api/resources.py

```diff
@@ -43,7 +43,7 @@ class Root(Resource):
     ]
 
 
-def create_root(_):
+def create_root(request):
```

Comment:
I'll remove it, but I find it awkward not to name the variable when we're forced to receive it.